operator/operator_test.go

```diff
@@ -114,11 +118,12 @@ type environment struct {
 	user2             *test.User
 	errg              *errors.Gatherer
 	enclaveParameters tee.Parameters
+	stopEnclave       func() error
```

Comment:
Do you ever call `stopEnclave` from the test? I'd also rename this field to `cleanup` because that's how you call the return function of `testSetup.setup`

Reply:
reverted all changes to the operator test.